ObservatoryCore/UI/Views/NotificationView.axaml.cs

```diff
@@ -10,11 +13,53 @@ public NotificationView()
         {
             InitializeComponent();
             SystemDecorations = SystemDecorations.None;
-            var screenBounds = Screens.Primary.Bounds;
-            Position = screenBounds.BottomRight - new PixelPoint((int)Width, (int)Height);
+
+            MakeClickThrough(); //Platform specific, currently windows only.
+            
+            int screen = Properties.Core.Default.NativeNotifyScreen;
+            int corner = Properties.Core.Default.NativeNotifyCorner;
+            string font = Properties.Core.Default.NativeNotifyFont;
+            
+            if (font.Length > 0)
+            {
+                var titleText = this.Find<TextBlock>("Title");
+                var detailText = this.Find<TextBlock>("Detail");
+                var fontFamily = new Avalonia.Media.FontFamily(font);
+
+                titleText.FontFamily = fontFamily;
+                detailText.FontFamily = fontFamily;
+            }
+
+            PixelRect screenBounds;
+
+            if (screen == -1)
+                screenBounds = Screens.Primary.Bounds;
+            else
+                screenBounds = Screens.All[screen - 1].Bounds;
+
+            double scale = LayoutHelper.GetLayoutScale(this);
+            double scaleWidth = Width * scale;
+            double scaleHeight = Height * scale;
+
+            switch (corner)
+            {
+                default: case 0: 
+                    Position = screenBounds.BottomRight - new PixelPoint((int)scaleWidth + 50, (int)scaleHeight + 50);
+                    break;
+                case 1:
+                    Position = screenBounds.BottomLeft - new PixelPoint(-50, (int)scaleHeight + 50);
+                    break;
+                case 2:
+                    Position = screenBounds.TopRight - new PixelPoint((int)scaleWidth + 50, -50);
+                    break;
+                case 3:
+                    Position = screenBounds.TopLeft + new PixelPoint(50, 50);
+                    break;
+            }
+
             var timer = new System.Timers.Timer();
             timer.Elapsed += CloseNotification;
-            timer.Interval = 5000;
+            timer.Interval = 8000;
```

Comment:
Consider a setting.

Reply:
Already considered! Will be adding one at some point for sure, might squeeze it into this PR, but I want to get a release out this weekend and am out of town most of today.